src/components/regions/ProbeDetails.svelte

```diff
@@ -22,8 +22,8 @@ const PROBE_TYPE_DOCS = {
 let visible = false;
 onMount(() => { visible = true; });
 
-function probeIsSelected(probe) {
-  return probe.name !== null && probe.name !== 'null';
+function probeIsSelected(p) {
```

Comment:
the other way should've been fine too but I really like this for human readability between this and the import, also consider `probeItem` or `probeObj`

Reply:
The original way raised an eslint warning. I agree `p` isn't the best name, so I'll update it. Thanks!